templates/zerver/api/deploying-bots.md

```diff
@@ -63,10 +63,10 @@ pip install zulip_botserver
 
 1. Run the Botserver, where `helloworld` is the name of the bot you
    want to run:
-   `zulip-bot-server --config-file <path_to_zuliprc> --bot-name=helloworld`
+   `zulip-run-botserver --config-file <path_to_zuliprc> --bot-name=helloworld`
```

Comment:
What's the thinking behind this rename?

Reply:
This rename is the twin of https://github.com/zulip/python-zulip-api/pull/411/commits/fd3b4a8cb7df109f3f25ffd69caa61b26f2824ed.

My reasoning is that now that we agreed on Botserver over bot server, or bot-server, it'd be best to remove the hyphen in zulip-**bot-server**. `zulip-botserver` would work, but since we already have a command `zulip-run-bot`, I thought of `zulip-run-botserver` to be the most consistent.

Reply:
I'd probably prefer `zulip-botserver`; I think `zulip-run-bot` is a bit cumbersome and not necessarily a convention we need to copy (though I don't think `zulip-bot` would be reasonable there)